public/styles/_docs.scss

```diff
@@ -1,209 +1,203 @@
 
 // Docs ------------------------------
 
-.docs-page {
-  a { color: #3a585f; }
-}
+.PRIMER-REMOVE-ME {
```

Comment:
Wrapped the entire file with `.PRIMER-REMOVE-ME` so the diff shows a lot of whitespace changes